### ISSUE TITLE
Use base class docstrings

### ISSUE THREAD

**Issue by mcgibbon** [CliMT/climt#45]
Currently there are many component docstrings on __init__, __call__ or array_call methods which don't really say much of anything useful that the base class docstrings don't already say. For those components, we should remove the docstring entirely so that Sphinx will use the base class docstrings (which are written to be generic enough to apply to any component). The important part of the documentation is present in the *_properties dictionaries of the components.

We also could think about a way to include those property dictionaries in the documentation, but that's a separate issue.

**Comment by JoyMonteiro** [comment]
In most components `__init__` always has quite detailed docstring. I agree that we should eliminate docstrings for `__call__` which don't really do much.

**Comment by mcgibbon** [comment]
Yes, there are just a couple `__init__` methods with no kwargs where I'll remove the docstring.
